std/traits.d

```diff
@@ -7921,6 +7941,26 @@ template getSymbolsByUDA(alias symbol, alias attribute)
     static assert(getSymbolsByUDA!(D,UDA).length == 0);
 }
 
+// Issue 18314
+@safe unittest
+{
+    enum lol;
+    enum lal;
+
+    struct A
+    {
+        @lal
```

Comment:
insert this declaration **first**

```d
@lal
void foo()(string){}
```

update the assertions accordingly, and have fun.

Reply:
For the sake of style, could we use `@Attr1` and `@Attr2` instead of `@lol` and `@lal`? I know I originally used those in my issue report, but it definitely was not my intention that such funny names ended up in the source code.

Reply:
This bug has already been filed in BugZilla. Looks to be by you: 
https://issues.dlang.org/show_bug.cgi?id=16206

It's not the mandate of this PR to fix bugs in DMD.

Reply:
Indeed. I planned to put the link to the issue after a possible "wtf" reaction from your part. But things didn't happen as expected.

Reply:
WTF?!
Here. Feeling better now?

Reply:
> I originally used those in my issue report, but it definitely was not my intention that such funny names ended up in the source code.

If you already come with the style, it should be `attr1` and `attr2` - https://dlang.org/dstyle.html#naming_udas

But meh, I don't have a preference between these two. It's quite typical for contributors to put the code from the issue 1:1 into Phobos. Also it doesn't end up in the user documentation.
And if you disagree, this can always be improved. It shouldn't block this PR.

Reply:
> Indeed. I planned to put the link to the issue after a possible "wtf" reaction from your part. But things didn't happen as expected.

Please refrain from wasting valuable time of other contributors in the future.
A simple comment:

> I just bumped into a similar issue. See XYZ

would have saved all parties time...

Reply:
Sure, but we'd have less fun that way.